docs/tutorials/trajectories.rst

```diff
@@ -0,0 +1,72 @@
+==========================
+Handle Expert Trajectories
+==========================
+
+For imitation learning we need trajectories.
+Some are generated by experts and serve as demonstrations,
+others are generated by the agent and serve as training data for a discriminator.
+In this library they are stored in a :class:`Trajectory <imitation.data.types.Trajectory>` dataclass:
+
+.. code-block:: python
+
+    @dataclasses.dataclass(frozen=True)
+    class Trajectory:
+        obs: np.ndarray
+        """Observations, shape (trajectory_len + 1, ) + observation_shape."""
+
+        acts: np.ndarray
+        """Actions, shape (trajectory_len, ) + action_shape."""
+
+        infos: Optional[np.ndarray]
+            """An array of info dicts, shape (trajectory_len, )."""
+
+        terminal: bool
+        """Does this trajectory (fragment) end in a terminal state?"""
+
+:class:`TrajectoryWithRew <imitation.data.types.TrajectoryWithRew>` is a subclass of
+:class:`Trajectory <imitation.data.types.Trajectory>` and has another
+:class:`rews <imitation.data.types.TrajectoryWithRew.rews>` field,
+which is an array of rewards of shape `(trajectory_len, )`.
+
+Usually, they are passed around as sequences of trajectories.
+
+Some algorithms do not care about entire trajectories, but only for individual
+:class:`Transitions <imitation.data.types.Transitions>` (:func:`flattened <imitation.data.rollout.flatten_trajectories>` trajectories).
```

Comment:
```suggestion
Some algorithms do not need as much information about the ordering of states, actions and rewards. Rather than using trajectories, these algorithms can make use of individual
:class:`Transitions <imitation.data.types.Transitions>` (:func:`flattened <imitation.data.rollout.flatten_trajectories>` trajectories).
```